raiden/routing.py

```diff
@@ -209,12 +210,15 @@ def get_best_routes_internal(
         # The complete route includes the initiator, add it to the beginning
         complete_route = [Address(from_address)] + neighbour.route
 
+        # https://github.com/raiden-network/raiden/issues/4751
+        # Internal routing doesn't know about fee so it should set a percentage per hop
+        estimated_fee = FeeAmount(round(INTERNAL_ROUTING_DEFAULT_FEE_PERC * amount))
```

Comment:
We're adding fees here even in case of a direct transfer.

Reply:
Sorry for not being thorough, yesterday. It also looks like we only apply `INTERNAL_ROUTING_DEFAULT_FEE_PERC` once. Wasn't the intention of the ticket to do that once per hop?

Reply:
I'll write a PR that changes these two things. If any of these statements is wrong, let me know!

Reply:
@karlb 

> We're adding fees here even in case of a direct transfer.

It's correct. I did not notice that.


> It also looks like we only apply INTERNAL_ROUTING_DEFAULT_FEE_PERC once. Wasn't the intention of the ticket to do that once per hop?

No, since we are not sure of the hops anyway in internal routing the intention (and it's also written in the issue) was to add a big percentage.